tests/dummy/app/pods/docs/guides/header/columns/template.md

```diff
@@ -184,3 +184,26 @@ reorder has occured.
 
   {{demo.snippet name='docs-example-resize-reorder-actions.hbs'}}
 {{/docs-demo}}
+
+## Text alignment
+
+A column can have its text aligned left, center or right by setting the `textAlign` property on the column definition.
+
+When the property is set, the cell will have the matching class (`text-align-left`, `text-align-center` or `text-align-center`).
```

Comment:
While @mixonic rudely was looking over my shoulder, he proposed to namespace classes to avoid collisions, something to the extent of `ember-table__text-align-left`

Reply:
Wouldn't that be something we do once consistently across the whole repo?

many classes do not have references to ember table (`is-sortable` for example) and some are prefixing with `et` (`et-header-resize-area`).

I chose `text-align-left` because it's what is used by ET1 ( https://github.com/Addepar/ember-table-legacy/blob/635316521c2f6e347a8b37fb75c972dd25f97bd5/addon/styles/addon.less#L63-L71), which allows for a simpler transition.

Reply:
🤷‍♂ This PR adds the styles. So this PR seems like a good place to not introduce a possible collision.

CSS class style compat with ET1 is a fine goal. Is that already something true? I thought the DOM changed so much that styles applied to ET1 are really not expected to work for ET2.

Reply:
It' not true for us as we don't customize based on the alignment. I don't know about other applications and how they scoped it (`.ember-table .text-align-center` would still work for example).

Happy to change as I don't think it's worth discussing that much, but it feels weird to add a 3rd style to name classes.